omnibus/config/software/datadog-agent-finalize.rb

```diff
@@ -66,12 +66,18 @@
             # remove unused configs
             delete "/etc/datadog-agent/conf.d/apm.yaml.default"
 
+            # remove windows specific configs
+            delete "#{install_dir}/etc/conf.d/winproc.d"
```

Comment:
we should remove `/etc/datadog-agent/conf.d/winproc.d` here, at this point we've already moved all of `#{install_dir}/etc/datadog-agent/conf.d` to `/etc/datadog-agent`